.github/workflows/sync_usersignup_crd.yml

```diff
@@ -0,0 +1,37 @@
+name: Sync User-signup CRD
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Run at 00:00 every Sunday
+
+jobs:
+  sync_file:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Fetch the file from toolchain repository and check for changes
+        id: fetch_and_check
+        run: |
+          EXTERNAL_FILE_URL="https://raw.githubusercontent.com/codeready-toolchain/host-operator/master/config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml"
+          TARGET_FILE_PATH="kwok/toolchain.dev.openshift.com_usersignups.yaml"
+          curl -o "$TARGET_FILE_PATH" "$EXTERNAL_FILE_URL"
+          if ! git diff --quiet "$TARGET_FILE_PATH"; then
+            echo "Changes detected."
+            echo "changes_detected=true" >> $GITHUB_ENV
+            git add "$TARGET_FILE_PATH"
+          else
+            echo "No changes detected. No action needed."
+          fi
+
+      - name: Commit and push changes
+        if: ${{ env.changes_detected == 'true' }}
+        run: |
+          git commit -m "Update local usersignup with toolchain repository"
+          git push
```

Comment:
Is it pushing directly to the branch? No PR?

Reply:
this file has already undergone a review from toolchain repo. will create a PR to keep us in track as well.